index.bs

```diff
@@ -980,15 +985,23 @@ partial interface HTMLIFrameElement {
     policy</a>, which may be empty.
     1. If |element| is not an <{iframe}> element, then return an empty [=policy
        directive=].
+    1. Let |allowAttribute| be the value of |element|'s <{iframe/allow}>
+       attribute.
     1. Let |container policy| be the result of running <a abstract-op>Parse policy
-      directive</a> given the value of |element|'s <{iframe/allow}> attribute,
-      the [=Document/origin=] of |element|'s [=node document=], and |element|'s
-      <a>declared origin</a>.
+      directive</a> given |allowAttribute|, the [=Document/origin=] of |element|'s
+      [=node document=], and |element|'s <a>declared origin</a>.
     1. If |element|'s <{iframe/allowfullscreen}> attribute is specified, and
        |container policy| does not [=map/contain=] an entry for the
        <code>fullscreen</code> [=policy-controlled feature|feature=].
       1. [=map/Set=] |container policy|[<code>fullscreen</code>] = <a>the
          special value <code>*</code></a>.
+    1. [=map/For each=] [=feature=] in |container policy|:
+      1. If [=feature=]'s <a>allowlist</a> does not [=list/contain=] <a>the special
+         value <code>*</code></a>, then [=list/for each=] [=origins=] in
+         <a>allowlist</a>:
+        1. Executing <a abstract-op>Is feature enabled in document for origin?</a>
```

Comment:
"Call" or "Execute" rather than "Executing"?

We might want a note here that the algorithm is being called specifically for the side effect of generating a report, not for actual usage of the feature.

Reply:
Done!